lib/ansible/modules/network/asa/asa_ogcontent.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: asa_ogcontent
+version_added: "2.10"
+author:
+    - "Federico Olivieri (@Federico87)"
+short_description: Find object-group' s contet.
+description:
+    - Find object-group' s contet including content of parent and child nested object-groups on Cisco ASA device.
+options:
+    name:
+        description:
+            - Name of the object-group.
+        required: true
+        type: str
+"""
+
+EXAMPLES = """
+---
+- name: find object-group's content.
+  asa_ogcontent:
+    name: aws_all_critical_vpcs
+"""
+
+RETURN = """
+commands:
```

Comment:
I don't know that `commands` is the right key for this. In other modules, `commands` indicates commands that were sent to the device to achieve a state. This module, as far as I can tell, is only informational in purpose (and should probably instead be part of a facts module?)

Reply:
@Qalthos you are right. That was my concern at the begin of this PR: I was not sure if there was a better way to implement this kind of module. I am ok to turn in a fact module but what would be the best way to do it? What would I need to change? Just a few indications would be great! Thanks